pkg/users/account-create-dialog.js

```diff
@@ -246,6 +265,7 @@ export function account_create_dialog(accounts, min_uid, max_uid, shells) {
         user_name: "",
         password: "",
         password_confirm: "",
+        password_confirm_dirty: false,
```

Comment:
I wouldn't know what "dirty" means in this context; Something like "_do_check" perhaps?